scripts/buildWeeklyRequestCountDocuments/index.js

```diff
@@ -0,0 +1,53 @@
+const admin = require('firebase-admin')
+const { initFirebaseAdmin } = require('../utils/initFirebaseAdmin.js')
+const { firebaseTimestampToMoment } = require('../../mixins/date/firebaseTimestampToMoment')
+const { getFormattedEndOfWeek } = require('../../mixins/date/getFormattedEndOfWeek')
+const { getFormattedStartOfWeek } = require('../../mixins/date/getFormattedStartOfWeek')
+
+initFirebaseAdmin()
+
+const buildRequestCountDocuments = async () => {
+  const weeklyRequestCreation = {}
+
+  try {
+    const allRequests = admin
+      .firestore()
+      .collectionGroup('requests')
+
+    const snapshots = await allRequests.get()
+
+    await Promise.all(
+      snapshots.docs.map(async (doc) => {
+        console.log(`Adding request: ${doc.id} to the count`)
+
+        const data = doc.data()
```

Comment:
Does this need an `await`? If it doesn't, remove the `async` from this function and the `Promise.all` surrounding it, since you're not doing any async work.